### ISSUE TITLE
Error on installation

### ISSUE THREAD

**Issue by idangozlan** [tugrul/node-mcrypt#70]
```
$ npm install mcrypt --save

> mcrypt@0.1.13 install myApp/node_modules/mcrypt
> node-gyp rebuild

  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/3-way.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/arcfour.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/blowfish.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/blowfish-compat.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/cast-128.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/cast-256.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/des.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/enigma.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/gost.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/loki97.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/panama.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/rc2.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/rijndael-128.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/rijndael-192.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/rijndael-256.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/safer64.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/safer128.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/saferplus.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/serpent.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/tripledes.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/twofish.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/wake.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/algorithms/xtea.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/modes/cbc.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/modes/cfb.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/modes/ctr.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/modes/ecb.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/modes/ncfb.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/modes/nofb.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/modes/ofb.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/modules/modes/stream.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/lib/bzero.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/lib/mcrypt.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/lib/mcrypt_extra.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/lib/mcrypt_modules.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/lib/mcrypt_symb.o
  CC(target) Release/obj.target/libmcrypt/lib/libmcrypt/lib/xmemory.o
  LIBTOOL-STATIC Release/mcrypt.a
  CXX(target) Release/obj.target/mcrypt/src/mcrypt.o
../src/mcrypt.cc:102:48: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
        return info.GetReturnValue().Set(cons->NewInstance(2, argv));
                                               ^
/Users/someuser/.node-gyp/6.11.4/include/node/v8.h:3271:3: note: 'NewInstance' has been explicitly marked deprecated here
  V8_DEPRECATED("Use maybe version",
  ^
/Users/someuser/.node-gyp/6.11.4/include/node/v8config.h:336:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
1 warning generated.
  SOLINK_MODULE(target) Release/mcrypt.node
clang: warning: libstdc++ is deprecated; move to libc++ with a minimum deployment target of OS X 10.9 [-Wdeprecated]
```

OS: MacOS High Sierra 10.13.3
Node: v6.11.4
NPM: 3.10.10

**Comment by klinkov** [comment]
Same issue

**Comment by tugrul** [comment]
This should be resolved in actual version (0.1.15). Please comment if is it not ok.
